CHANGES_UNRELEASED.md

```diff
@@ -3,3 +3,15 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.34.0...master)
+
+## FxA Client
+
+### Breaking changes
+
+- The `FirefoxAccount.destroyDevice` method has been removed in favor of the
+  more general `FirefoxAccount.disconnect` method which will ensure a full
+  disconnection by invalidating OAuth tokens and destroying the device record
+  if it exists. ([#1397](https://github.com/mozilla/application-services/issues/1397))
+- The `FirefoxAccount.disconnect` method has been added to the Swift bindings as well.
+- The `FirefoxAccount.beginOAuthFlow` method will redirect to a content page that
+  forces the user to connect to the last seen user email.
```

Comment:
nit: is it worth adding that if you *don't* want this behaviour, you need to `disconnect()` explicitly?

Reply:
(At least, that's what I took away from previous discussion; maybe they instead have to create a fresh `FirefoxAccount` instance from scratch?)

Reply:
Yup you got to throw away the instance, since `disconnect` puts you in a "re-connectable" state.